thread/thread.go

```diff
@@ -202,7 +204,12 @@ func (t *Thread) cleanUpOnLowDiskSpace() {
 			return
 		}
 		v(0, "Thread %v disk usage is high (packet path=%q): %d%% free <= %d%% threshold", t.id, t.packetPath, df, t.conf.DiskFreePercentage)
-		t.deleteOldestThreadFiles(1)
+		if lastdf > df {
+			delFileCnt *= 2
+			v(0, "Thread %v file deletion didn't surpass write speed. Increasing file deletion rate to %d", t.id, delFileCnt)
+		}
```

Comment:
I thought about this some more and I couldn't come up with an accurate way to calculate the ramp down without increasing the precision of base.PathDiskFreePercentage  (float64?)  I've got a commit mostly thrown together that I can send if you think this is best way.